src/entry-points/popup/App.svelte

```diff
@@ -609,179 +530,64 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
-      >
-      <span>{getMessage('enable')}</span>
-    </label>
-  </div>
-  <!-- TODO but this is technically a button. Is this ok? -->
-  <button
-    id="options-button"
-    on:click={() => {
-      browserOrChrome.runtime.openOptionsPage();
-      if (isMobile) {
-        // The options tab gets opened, but it's not visible
-        // because the popup stays open. Let's close it.
-        window.close();
-      }
-    }}
-    use:tippy={{
-      content: () => getMessage('more'),
-      theme: 'my-tippy',
-    }}
-  >⚙️</button>
-  <div class="others__wrapper">
-    {#if settings.advancedMode}
-    <!-- TODO work on accessibility for the volume indicator. https://atomiks.github.io/tippyjs/v6/accessibility. -->
-    <span
-      class="others__item"
-      use:tippy={{
-        content: () => {
-          let tooltip = getMessage('volume');
-          const hotkeysString =
-            getActionString(
-              HotkeyAction_INCREASE_VOLUME,
-              getMessage("increaseSettingValue")
-            )
-            + getActionString(
-              HotkeyAction_DECREASE_VOLUME,
-              getMessage("decreaseSettingValue")
-            );
-          if (hotkeysString) {
-            tooltip += '\n' + hotkeysString;
-          }
-
-          return tooltip;
-        },
-        theme: tippyThemeMyTippyAndPreLine,
-      }}
-    >
-      <!-- `min-width` because the emojis have different widths, so it remains constant. -->
-      <span
-        style="display: inline-block; min-width: 2.5ch;"
-      >{(() => {
-        if (!latestTelemetryRecord) return '🔉';
-        const vol = latestTelemetryRecord.elementVolume;
-        if (vol < 0.001) return '🔇';
-        if (vol < 1/3) return '🔈';
-        if (vol < 2/3) return '🔉';
-        return '🔊';
-      })()}</span>
-      <!-- TODO how about we replace it with a range input. -->
-      <meter
-        min="0"
-        max="1"
-        value={latestTelemetryRecord?.elementVolume ?? 0}
-        style="width: 6rem;"
-      ></meter>
-    </span>
-    {:else}
-    <div style="flex-grow: 1"></div>
-    {/if}
-
-    <!-- TODO in "simple mode" the toolip's content is cut clipped. -->
-    <!-- Why button? So the tooltip can be accessed with no pointer device. Any better ideas? -->
-    <button
-      type="button"
-      class="others__item"
-      style="border: none; padding: 0; background: unset; font: inherit;"
-      use:tippy={{
-        content: timeSavedTooltipContentEl,
-        theme: 'my-tippy',
-        placement: 'bottom',
-        hideOnClick: false,
-      }}
-    >
-      <span>⏱️</span>
-      <span>{timeSavedPlaybackRateEquivalentsFmt[0]}</span>
-      {#if settings.timeSavedAveragingMethod !== 'exponential'}
-        <span>({timeSavedComparedToSoundedSpeedAbs} / {wouldHaveLastedIfSpeedWasSounded})</span>
+  <div style="display: flex; justify-content: space-between;">
+    <div>
+      <div style="margin-bottom: 5px;">
```

Comment:
I think row wrappers would make more sense, as opposed to column wrappers. Also ensures that things don't get ugly on different DPIs.